### PR TITLE
TT-264 UI test coverage

### PR DESCRIPTION
**Pull request by kevinjlope**
In this PR, I **added** some unit testing in the methods that were not tested, and  I **removed** the unit testing that was unnecessary in the project.

### CODE REVIEW

File: src/app/modules/customer-management/components/projects/components/project-list/project-list.component.spec.ts [hunk 1]
```diff
@@ -86,10 +85,20 @@ describe('ProjectListComponent', () => {
   it('should destroy the subscriptions', () => {
     component.projectsSubscription = new Subscription();
     const subscription = spyOn(component.projectsSubscription, 'unsubscribe');
-
     component.ngOnDestroy();
 
     expect(subscription).toHaveBeenCalledTimes(1);
+
```
Comment (scastillo-jp) [inline src/app/modules/customer-management/components/projects/components/project-list/project-list.component.spec.ts:91]: dete this line white

File: src/app/modules/customer-management/components/projects/components/project-list/project-list.component.spec.ts [hunk 2]
```diff
@@ -86,10 +85,20 @@ describe('ProjectListComponent', () => {
...
+  });
+
+  it('should destroy the projectTypesSubscription', () => {
+    component.projectTypesSubscription = new Subscription();
+    const subscriptionProjectTypes = spyOn(component.projectTypesSubscription, 'unsubscribe');
+
+    component.ngOnDestroy();
+
+    expect(subscriptionProjectTypes).toHaveBeenCalledTimes(1);
+
```
Comment (scastillo-jp) [inline src/app/modules/customer-management/components/projects/components/project-list/project-list.component.spec.ts:101]: delete this line white

File: src/app/modules/customer-management/components/projects/components/project-list/project-list.component.spec.ts [hunk 3]
```diff
@@ -136,6 +144,7 @@ describe('ProjectListComponent', () => {
     spyOn(component, 'openModal');
     component.switchStatus(itemData);
     expect(component.openModal).toHaveBeenCalled();
+
```
Comment (scastillo-jp) [inline src/app/modules/customer-management/components/projects/components/project-list/project-list.component.spec.ts:147]: delete this line white

File: src/app/modules/user/services/user.service.spec.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import { TestBed, inject } from '@angular/core/testing';
+import { UserService } from './user.service';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+
+
```
Comment (scastillo-jp) [inline src/app/modules/user/services/user.service.spec.ts:5]: delete this line-white

**Comment by scastillo-jp** [comment]
@mrkevinomar  in my localist host using your branch I have problems with the test:
**should preload data of last entry when a project is selected while creating new entry**
`Error: Expected $.start_date = Date(Tue Jun 29 2021 14:16:46 GMT-0500 (Ecuador Time)) to equal Date(Tue Jun 29 2021 14:16:46 GMT-0500 (Ecuador Time)).`

**Review by scastillo-jp — APPROVED** [review]

**Review by jase156 — APPROVED** [review]

**Review by scastillo-jp — APPROVED** [review]

**Pull request merged**